Frameworks/Foundation/NSMutableOrderedSet.mm

```diff
@@ -277,4 +285,11 @@ - (void)unionSet:(NSSet*)other {
     UNIMPLEMENTED();
 }
 
+/**
+ @Status Interoperable
+*/
+- (id)copyWithZone:(NSZone*)zone {
+    return [[NSMutableOrderedSet alloc] initWithOrderedSet:self];
```

Comment:
Oops. I'm a dummy and forgot about the mutableCopy guys. Carry on. 

---
In reply to: [64486017](https://github.com/Microsoft/WinObjC/pull/482#discussion_r64486017) [](ancestors = 64486017)